
package exception;

public class Test {
    public static void main(String [] args)
    {
        int i=0;
        int j=324;
        int k=9;
        try
        {
        j=k/i;
        }
        catch(Exception e)
        {
            System.out.println("tui ki gadhar baccha?");
        }
            
        }
    
}
